### PR TITLE
Remove qt4 sans

### PR DESCRIPTION
**Pull request by DavidFair**
**Description of work.**

Removes MantidPlot imports and calls from the SANS codebase
In the majority of cases I've replaced them with NotImplementedErrors either because the module is deprecated, or because it's been broken since Mantid 6.0 without users having noticed it.

The intention is that instead of silently not plotting anything we explicitly tell users it's not working with workbench please contact us if you need this.

**To test:**
- Download ISIS Training data
- Ensure LOQDemo folder is present
- Open ISIS SANS interface
- Load a .txt or .toml file from the LOQ Demo folder
- Enter 74044 into the left hand column
- Hit load
- Go to Settings, Mask, ensure row 0 is selected (hit update rows if not)
- Display mask should open the instrument viewer

There is no associated issue. - This was noticed whilst working on plotting code

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by gemmaguest — CHANGES_REQUESTED** [review]
Code changes look good and works as expected :+1: Just a couple of very minor suggestions

**Review by gemmaguest — APPROVED** [review]

File: scripts/SANS/sans/command_interface/ISISCommandInterface.py [hunk 1]
```diff
@@ -1020,8 +1013,8 @@ def PhiRanges(phis, plot=True):
         reduced_workspace_name = WavRangeReduction()
         reduced_workspace_names.append(reduced_workspace_name)
 
-    if plot and mantidplot:
-        mantidplot.plotSpectrum(reduced_workspace_names, 0)
+    if plot:
+        raise NotImplementedError("Plotting is not implemented for workbench yet, please contact the dev team.")
```
Comment (gemmaguest) [inline scripts/SANS/sans/command_interface/ISISCommandInterface.py:1017]: Same here regarding the forum

File: scripts/SANS/sans/command_interface/ISISCommandInterface.py [hunk 2]
```diff
@@ -1076,6 +1069,8 @@ def PlotResult(workspace, canvas=None):
         @param canvas: optional handle to an existing graph to write the plot to
         @return: a handle to the graph that was written to
     """
+    raise NotImplementedError("Plotting is not implemented for workbench yet, please contact the dev team.")
```
Comment (gemmaguest) [inline scripts/SANS/sans/command_interface/ISISCommandInterface.py:1072]: Same here regarding the forum

**Review by gemmaguest — CHANGES_REQUESTED** [review]
Just a couple more places we should mention the forum!

File: scripts/SANS/sans/command_interface/ISISCommandInterface.py (hunk 2 above)
Comment (DavidFair) [reply scripts/SANS/sans/command_interface/ISISCommandInterface.py]: The worst part is I thought, "I'm sure I've written this more than once", then promptly didn't check :face_with_head_bandage:

**Review by gemmaguest — APPROVED** [review]

**Pull request merged**